### PR TITLE
Adjust shield padding for Roboto Condensed metrics

### PR DESCRIPTION
**Pull request by claysmalley**
Small adjustments to the padding of a bunch of shield icons to make sure both Arial Narrow and Roboto Condensed choose the right width variant icon, and otherwise look good.

### CODE REVIEW

File: src/js/screen_gfx.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 "use strict";
 
-const fontFamily = '"sans-serif-condensed", "Arial Narrow", sans-serif';
+const fontFamily =
+  '"sans-serif-condensed", "Arial Narrow", "Roboto Condensed", "Noto Sans Armenian", sans-serif';
 export const shieldFont = (size) => `bold ${size}px ${fontFamily}`;
```
Comment (jleedev) [inline src/js/screen_gfx.js:5]: Add this and it actually works on firefox android

```suggestion
export const shieldFont = (size) => `condensed bold ${size}px ${fontFamily}`;
```

**Comment by claysmalley** [comment]
Also a subtle change to make rounded rectangle shield horizontal padding match that of oval shields.

before/after: 
Screenshot from 2022-08-20 11-31-16 Screenshot from 2022-08-20 11-31-29

**Comment by claysmalley** [comment]
Converting to draft pending merge of #544.

**Comment by claysmalley** [comment]
Mostly obsoleted by new draw functions.

**Pull request closed without merging**